src/index.html

```diff
@@ -11,6 +11,17 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#site" class="form form--01"
+     method="post">
```

Comment:
in this case GET will be correct, also you should use uppercase when it comes to method attribute